src/TTextEdit.cpp

```diff
@@ -1019,7 +1019,7 @@ void TTextEdit::mousePressEvent(QMouseEvent* event)
             mudletEvent.mArgumentList.append(QString::number(3));
             break;
         default: // TODO: What about those of us with more than three mouse buttons?
-            mudletEvent.mArgumentList.append(0);
+            mudletEvent.mArgumentList.append(nullptr);
```

Comment:
I think this is wrong I believe it should be `mudletEvent.mArgumentList.append(QString::number(0));` or all of them could instead be of the form `mudletEvent.mArgumentList.append(QLatin1String("0"));` with `"0"` replaced by `"1"` etc...